dashboard/client/src/pages/dashboard/Dashboard.tsx

```diff
@@ -147,11 +147,11 @@ const Dashboard: React.FC = () => {
             run the following command: `ray disable-usage-stats` before starting
             the cluster. See{" "}
             <a
-              href="https://github.com/ray-project/ray/issues/20857"
+              href="https://docs.ray.io/en/master/cluster/usage-stats.html"
```

Comment:
maybe `latest`?

Reply:
It's a new doc so before 1.13 is released, it will be invalid if we use `latest`. Also it's more about privacy policy for usage stats so its good to always point to master.